homeassistant/components/bang_olufsen/config_flow.py

```diff
@@ -118,45 +127,22 @@ async def async_step_zeroconf(
         self._serial_number = discovery_info.properties[ATTR_SERIAL_NUMBER]
         self._beolink_jid = f"{discovery_info.properties[ATTR_TYPE_NUMBER]}.{discovery_info.properties[ATTR_ITEM_NUMBER]}.{self._serial_number}@products.bang-olufsen.com"
 
-        self.context["title_placeholders"] = {
-            "name": discovery_info.properties[ATTR_FRIENDLY_NAME]
-        }
+        return await self._create_entry()
```

Comment:
We need a confirm step for devices discovered via zeroconf. If you search the codebase for `async_step_zeroconf_confirm` you will find many examples 👍

Reply:
Ah. so I did it correctly the first time. I've added a confirm step for zeroconf discovery.

Reply:
Yes, I'm sorry for the confusing review comments previously.